src/pages/public_registration/ChapterPublisherInfo.tsx

```diff
@@ -15,22 +15,20 @@ interface ChapterPublisherInfoProps {
 export const ChapterPublisherInfo = ({ publicationContext }: ChapterPublisherInfoProps) => {
   const { t } = useTranslation();
 
-  const publisherIdentifier = publicationContext.id && getIdentifierFromId(publicationContext.id);
+  const identifier = publicationContext.id ? getIdentifierFromId(publicationContext.id) : '';
 
   const publisherQuery = useQuery({
-    enabled: !!publisherIdentifier,
-    queryKey: ['publisher', publisherIdentifier],
-    queryFn: () => fetchResults({ id: publisherIdentifier, results: 1 }),
+    enabled: !!identifier,
+    queryKey: ['publisher', identifier],
```

Comment:
burde kanskje bruke `registration` som key, som vi stort sett bruker ellers når vi bruker `fetchRegistration`? Større sjans for å kunne bruke en cached verdi
```suggestion
    queryKey: ['registration', identifier],
```